SparkPost/2web2ui
# SA-1335: Updating chart styling for hibana

### What Changed
 - Updating charts + tooltip to match mocks and new hibana style

### How To Test
 - On an account with Hibana enabled, open Report builder and access the chart
 - Select several various metrics
 - Check that styles match correctly

### Notes
- Bar used as a background. We could try searching for max value, but that seems like expensive operation
- Bar doesn't have background hover operation
